packages/flutter/lib/src/services/text_input.dart

```diff
@@ -2589,29 +2598,96 @@ class SystemContextMenuController with SystemContextMenuClient {
     );
 
     // Don't show the same thing that's already being shown.
-    if (_lastShown != null && _lastShown!._isVisible && _lastShown!._lastTargetRect == targetRect) {
+    if (_lastShown != null && _lastShown!.isVisible && _lastShown!._lastTargetRect == targetRect) {
       return Future<void>.value();
     }
 
     assert(
-      _lastShown == null || _lastShown == this || !_lastShown!._isVisible,
+      _lastShown == null || _lastShown == this || !_lastShown!.isVisible,
       'Attempted to show while another instance was still visible.',
     );
 
+    ServicesBinding.systemContextMenuClient = this;
+
     _lastTargetRect = targetRect;
     _lastShown = this;
     _hiddenBySystem = false;
-    return _channel.invokeMethod<Map<String, dynamic>>(
-      'ContextMenu.showSystemContextMenu',
-      <String, dynamic>{
-        'targetRect': <String, double>{
-          'x': targetRect.left,
-          'y': targetRect.top,
-          'width': targetRect.width,
-          'height': targetRect.height,
-        },
+    return _channel.invokeMethod('ContextMenu.showSystemContextMenu', <String, dynamic>{
+      'targetRect': <String, double>{
```

Comment:
We don't need to pass the default list of items here?

Reply:
I just changed this (see https://github.com/flutter/flutter/pull/159013#issuecomment-2634939771).

Now there are two methods: `show`, which is deprecated, and `showItems`. The old method doesn't take or pass `items` at all, and iOS shows a default set of buttons. The new method requires `items` to be non-empty. The default items are set at the widget level, because it needs EditableTextState to determine what they are (e.g. don't show the Copy button if the selection is collapsed).

Reply:
> The reason why we don't want to let the platform choose the items any more is that the platform includes a button that Flutter doesn't support, Autofill. Pressing it does nothing.

Does this apply to the `show` method?

Reply:
Yes, calling `show` will show the Autofill button. For `showWithItems` it's not possible to make Autofill appear.